js/controllers/search.js

```diff
@@ -7,6 +7,9 @@ function SearchController(search) {
   $('#search-button').click(this.onSearchButton_.bind(this));
   $('#search-input').bind('input', this.onChange_.bind(this));
   $('#search-input').keydown(this.onKeydown_.bind(this));
+  // Prevent search deactivation when search count is clicked
```

Comment:
Can you try modifying deactivateSearch instead? Perhaps the thing you can look for is if e.composedPath.contains(this.searchContainer)?

It may be worthwhile caching search container for the lifetime of this class well, just a thought :)

Reply:
If this doesn't work, then what you have at the moment might be the right way to go :)

Reply:
Since this is a focusout event, the composedPath contains the ancestor line of the element *losing* focus, where we need to know when the element that has been clicked on is the search counter. I originally wanted to do this using e.relatedTarget (which for focusout event is the target the focus is moving to). Unfortunately for elements that can't receive focus (like this lowly span) e.relatedTarget is null :( 

Do you have any other ideas of how we can avoid preventDefault?

Also re: the caching comment, I think the lifetime of this class is the lifetime of the app?

Reply:
What about `this.searchContainer.contains(e.target)`?

Reply:
on focusout e.target would be the thing that focus is *leaving*. i.e. this.searchContainer.contains(e.target) would always be true whenever search is active and you click anything, regardless of whether the thing you have clicked is inside the search container or not.

the fact that you can't get any information about what is being clicked on when you click an element that can't receive focus means that you can't possibly differentiate between clicking on the serach count div or any other non focussable div outside the search box